rmw_connextdds_common/src/common/rmw_context.cpp

```diff
@@ -218,6 +238,24 @@ rmw_context_impl_t::initialize_node(
     return RMW_RET_ERROR;
   }
 
+  if (this->qos_library != "") {
+    struct DDS_StringSeq library_names;
+    DDS_StringSeq_initialize(&library_names);
```

Comment:
It should be sufficient to use `DDS_SEQUENCE_INITIALIZER`, instead of `Seq_initialize()`, e.g.:
```
struct DDS_StringSeq library_names = DDS_SEQUENCE_INITIALIZER;
```

Reply:
Done in https://github.com/rticommunity/rmw_connextdds/pull/7/commits/ffd3052d914604d3a783bd5c322a50b3263fdf61